_literate/4_Turing.jl

```diff
@@ -89,7 +89,7 @@ using Plots, StatsPlots, Distributions, LaTeXStrings
 dice = DiscreteUniform(1, 6)
 plot(dice,
     label="six-sided Dice",
-    ms=5,
+    lw=5,
```

Comment:
That's not what I intended. I want do a lollipop graph like in this PR: https://github.com/JuliaPlots/StatsPlots.jl/pull/451

See the images. I am looking for `markersize` (`ms`) not `linewidth` (`ls`).

Can you fix it to be similar to the PR I've shown above?

Reply:
ah! sure - like this?